warp10/src/main/java/io/warp10/worf/WorfInteractive.java

```diff
@@ -25,6 +25,7 @@
 import jline.console.completer.StringsCompleter;
 import org.apache.commons.lang.math.NumberUtils;
 import org.apache.commons.math3.util.Pair;
+import org.boon.core.Sys;
```

Comment:
Where is boon used?

Reply:
boon unused IDE auto import / line removed